en/restore-from-aws-s3-using-br.md

```diff
@@ -6,17 +6,24 @@ aliases: ['/docs/tidb-in-kubernetes/dev/restore-from-aws-s3-using-br/']
 
 # Restore Data from S3-Compatible Storage Using BR
 
-This document describes how to restore the TiDB cluster data backed up using TiDB Operator in Kubernetes. [BR](https://pingcap.com/docs/stable/br/backup-and-restore-tool/) is used to perform the restore.
+This document describes how to restore the TiDB cluster data backed up using TiDB Operator in Kubernetes.
 
-The restore method described in this document is implemented based on Custom Resource Definition (CRD) in TiDB Operator v1.1 or later versions.
+The restore method described in this document is implemented based on CustomResourceDefinition (CRD) in TiDB Operator. For the underlying implementation, [BR](https://docs.pingcap.com/tidb/stable/backup-and-restore-tool) is used to restore the data. BR stands for Backup & Restore, which is a command-line tool for distributed backup and recovery of the TiDB cluster data.
 
-This document shows an example in which the backup data stored in the specified path on the Amazon S3 storage is restored to the TiDB cluster.
+## User Scenarios
 
-## Prerequisites
+After backing up TiDB cluster data to Amazon S3 using BR, if you need to recover the backup SST (key-value pairs) files from Amazon S3 to TiDB cluster, you can follow steps in this document to restore the data using BR.
 
 > **Note:**
 >
-> If TiDB Operator >= v1.1.10 && TiDB >= v4.0.8, BR will automatically adjust `tikv_gc_life_time`. You do not need to configure `spec.to` fields in the `Restore` CR. In addition, you can skip the steps of creating the `restore-demo2-tidb-secret` secret and [configuring database account privileges](#required-database-account-privileges).
+> - BR is only applicable to TiDB v3.1 or later releases.
+> - Data restored by BR cannot be replicated to a downstream cluster, because BR imports SST files to TiDB directly and the downstream cluster currently cannot access the upstream SST files.
```

Comment:
```suggestion
> - Data restored by BR cannot be replicated to a downstream cluster, because BR directly imports SST files to TiDB and the downstream cluster currently cannot access the upstream SST files.
```